hadoop-ozone/dev-support/checks/_mvn_unit_report.sh

```diff
@@ -16,6 +16,15 @@
 
 REPORT_DIR=${REPORT_DIR:-$PWD}
 
+_realpath() {
+  if realpath "$@" > /dev/null; then
+    realpath "$@"
+  else
+    local relative_to=$(realpath "${1/--relative-to=/}")
```

Comment:
shellcheck:11: warning: Declare and assign separately to avoid masking return values. [SC2155]